### PR TITLE
[gno/en] Add Gno Tutorial

### PR DESCRIPTION
**Pull request by leohhhn**
Hello, we would like to add the Learn X in Y tutorial for [Gno](https://docs.gno.land). 

- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

cc @zivkovicmilos @michelleellen

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]

File: gno.html.markdown [hunk 1]
```diff
@@ -0,0 +1,533 @@
+---
+language: Gno
```
Comment (vendethiel) [inline gno.html.markdown:2]: you're missing the `filename` metadata field

Comment (leohhhn) [reply gno.html.markdown:2]: Hey @vendethiel, thanks for the comment.

I wasn't fully sure what that line was for, as the repo currently contains mainly html markdown files, without any other files. Can you clarify what this adding this line would do? 🙏🏻

Comment (vendethiel) [reply gno.html.markdown:2]: Sets the filename for the `Get the code:` button

**Comment by verhovsky** [comment]
I don't think this language should be included on Learn X in Y Minutes.

**Pull request closed without merging**